documentation/course/e2/README.md

```diff
@@ -1157,25 +1282,31 @@ Zgodnie z mechanizmem [`M/10`](#m10-relacyjne-bazy-danych-acid-na-rds), wszystki
 
 Zdecydowano się na wykorzystanie silnika **PostgreSQL** do wszystkich relacyjnych baz systemu, ze względu na jego popularność, znajomość w zespole, wsparcie na AWS RDS oraz licencję open-source. Wykorzystano najnowszą stabilną wersję **17.2**, która zapewnia najnowsze funkcje i poprawki bezpieczeństwa. PostgreSQL nie oferuje wersji LTS, natomiast każda wersja jest wspierana przez co najmniej 5 lat[^postgres-version].
 
-W przypadku klas instancji AWS RDS, wybrano najnowszą dostępną generację modeli ogólnego przeznaczenia **`db.t4g.*`**, polecaną przez AWS jako dobry domyślny wybór. Oferowane w zakresie generacji klasy różnią się głównie liczbą vCPU oraz dostępną pamięcią RAM[^rds-instance-types]:
+W przypadku klas instancji AWS RDS, wybrano najnowszą dostępną generację modeli ogólnego przeznaczenia **`db.m7g.*`**, polecaną przez AWS jako dobry domyślny wybór. Oferowane w zakresie generacji klasy różnią się głównie liczbą vCPU oraz dostępną pamięcią RAM[^rds-instance-types]:
 
-| **Model**            | **vCPU** | **Pamięć (GiB)** |
-| -------------------- | -------- | ---------------- |
-| **`db.t4g.micro`**   | 2        | 1                |
-| **`db.t4g.small`**   | 2        | 2                |
-| **`db.t4g.medium`**  | 2        | 4                |
-| **`db.t4g.large`**   | 2        | 8                |
-| **`db.t4g.xlarge`**  | 4        | 16               |
-| **`db.t4g.2xlarge`** | 8        | 32               |
+| **Model**             | **vCPU** | **Pamięć (GiB)** |
+| --------------------- | -------- | ---------------- |
+| **`db.m7g.large`**    | 2        | 8                |
+| **`db.m7g.xlarge`**   | 4        | 16               |
+| **`db.m7g.2xlarge`**  | 8        | 32               |
+| **`db.m7g.4xlarge`**  | 16       | 64               |
+| **`db.m7g.8xlarge`**  | 32       | 128              |
+| **`db.m7g.12xlarge`** | 48       | 192              |
+| **`db.m7g.16xlarge`** | 64       | 256              |
 
 Jako model składowania dla wszystkich RDS wybrano **`gp3`**, który jest najnowszym i rekomendowanym przez AWS typem generalnego przeznaczenia.
 
 W przedstawionych poniżej diagramach bazodanowych zastosowano następującą notację:
+
 - `PK` w lewej kolumnie - klucz główny,
 - `FK` w lewej kolumnie - klucz obcy,
 - `*` w lewej kolumnie - pole wymagane (brak gwiazdki oznacza pole opcjonalne),
+- podkreślenie nazwy - unikalna wartość,
 - liczności powiązań oznaczone poprzez notację [crow's foot](https://vertabelo.com/blog/crow-s-foot-notation/).
 
+> [!NOTE]
+> Przedstawiona tutaj decyzja zakłada brak jakichkolwiek ograniczeń. Ze względu na tworzenie tego projektu pod pewnymi ograniczeniami zaproponowane rozwiązanie będzie się różniło od późniejszego etapu wdrożenia. Ze względu na dostępne dla nas zasoby.
+
```

Comment:
As before, maybe add that these restrictions are due to the use of a student account?